### PR TITLE
creat table

### PR DESCRIPTION
**Pull request by Yemisirach**
- [ ] Create a database named vet_clinic_database.
- [ ] Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- [ ] Create a table named animals
- [ ] Copy the SQL you used in the previous point into the schema.sql, data.sql, and queries.sql

These are the results of the queries:
Screenshot1
Screenshot2
Screenshot3
Screenshot4

### CODE REVIEW

**Review by andersonlebon — COMMENTED** [review]
## Hi @Yemisirach 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional Readme file kindly follow this [microverse template](https://github.com/microverseinc/readme-template) and work on customizing your README file the request another review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @Yemisirach ✋

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Could you remove the entire live demo section from your readme file since this project is not deployed? This is a general requirement for all projects.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 
 CREATE TABLE animals (
-    name varchar(100)
+    id              INT,
+    name            CHAR(20),
+    date_of_birth   DATE,
+    escape_attempts INT, 
+    neutered        BOOLEAN,
+    weight_kg       DECIMAL
 );
```
Comment (andersonlebon) [inline schema.sql:10]: ## [OPTIONAL]

- [ ] Nice work so far 👏 , however, you can improve this by making the id column not null and autoincremented. Hint 👇 

```.sql
    id int NOT NULL IDENTITY(1,1)
```

**Review by andersonlebon — APPROVED** [review]
## Hi @Yemisirach 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- Good PR description :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**